### ISSUE TITLE
Tabs not showing correctly with most of themes

### ISSUE THREAD

**Issue by felipeprov** [json-editor/json-editor#121]
### Expected behavior
Show the content

### Actual behavior
The second tab keeps hidden

### Steps to reproduce the behavior
You can use this schema on the demo page
- open https://json-editor.github.io/json-editor/
- paste the schema
- change theme to bootstrap 4(i just saw it working on bootstrap2)
- the first tab is ok, but the second tab shows nothing

schema
```javascript
{
    "title": "Game",
    "type": "object",
    "format": "categories",
    "options": {
        "collapsed": true
    },
    "properties": {
      "terms":  {
          "type": "array",
          "format": "table",
          "title": "Termos",
          "uniqueItems": true,
          "items": {
              "type": "object",
              "title": "Termo",
              "properties": {
                  "id": {
                      "title": "ID (sem espaço sem acentos)",
                      "type":"string"
                  },
                  "value": {
                    "title": "Valor",
                    "type": "string",
                    "format": "html"
                  }
              }
          }
      },
      "scenes": {
          "type": "array",
          "title": "Cenas", 
          "uniqueItems": true, 
          "items": {
              "type": "object",
              "title": "Cena",
              "properties": {
                  "name": {
                      "type": "string"
                  },
                  "objects" : {
                      "type": "array",
                      "format":"table",
                      "title": "Objetos",
                      "items":{
                          "type": "object",
                          "title": "Objeto",
                          "properties": {
                              "name": {
                                  "type": "string"
                              },
                              "url": {
                                  "type": "string"
                              },
                              "x": {
                                  "type": "number"
                              },
                              "y": {
                                  "type": "number"
                              },
                              "scaleX": {
                                  "type": "number"
                              },
                              "scaleY": {
                                  "type": "number"
                              },
                              "rotation": {
                                  "type": "number"
                              }
                          }
                      } 
                  }
              }
          }
      }
    }
}
```

default values
```javascript
{
  "terms": [
    {
      "id": "TITLE",
      "value": "Title"
    }
  ],
  "scenes": [
    {
      "name": "TitleScene",
      "objects": [
        {
          "name": "HUB0",
          "url": "url1",
          "x": 500,
          "y": 300,
          "scaleX": 1,
          "scaleY": 1,
          "rotation": 0
        },
        {
          "name": "HUB1",
          "url": "url2",
          "x": 500,
          "y": 300,
          "scaleX": 1,
          "scaleY": 1,
          "rotation": 0
        }
      ]
    }
  ]
}
```

**Comment by dy93** [comment]
#103 should fix the tabs on bootstrap 3

**Comment by pmk65** [comment]
See #214

**Comment by pmk65** [comment]
Fixed in #243
